expat.spec

```diff
@@ -1,6 +1,9 @@
-### RPM external expat 2.1.0
+### RPM external expat 2.4.6
 ## INITENV +PATH LD_LIBRARY_PATH %{i}/lib64
-Source: http://downloads.sourceforge.net/project/%{n}/%{n}/%{realversion}/%{n}-%{realversion}.tar.gz
+%define uversion %(echo %v | sed -e 's/\\./_/g')
+Source: https://github.com/libexpat/libexpat/releases/download/R_%{uversion}/%{n}-%{v}.tar.gz
```

Comment:
please use `%{n}-%{realversion}.tar.gz` here . `%{v}` the version with contains the checksum too e.g. it will be `2.4.6-checksum`. `realversion` is the version mentioned in the first line of the spec